frontends/web/src/components/transactions/transactions.tsx

```diff
@@ -15,72 +15,64 @@
  * limitations under the License.
  */
 
-import { Component } from 'react';
+import { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
 import { ITransaction } from '../../api/account';
 import A from '../../components/anchor/anchor';
-import { translate, TranslateProps } from '../../decorators/translate';
 import { runningInAndroid } from '../../utils/env';
 import { Transaction } from './transaction';
 import style from './transactions.module.css';
 
-interface TransactionsProps {
+type Props = {
     accountCode: string;
     explorerURL: string;
     transactions?: ITransaction[];
     handleExport: () => void;
-}
+};
 
-type Props = TransactionsProps & TranslateProps;
+export const Transactions: FunctionComponent<Props> = ({
+  accountCode,
+  explorerURL,
+  transactions,
+  handleExport,
+}) => {
+  const { t } = useTranslation();
+  const csvExportDisabled = runningInAndroid();
```

Comment:
I noticed that you didn't include the comment about Android CSV export. Was it on purpose?